include/cuco/detail/storage/kernels.cuh

```diff
@@ -19,37 +19,29 @@
 
 #include <cstddef>
 
-namespace cuco {
-namespace detail {
+namespace cuco::detail {
 
 CUCO_SUPPRESS_KERNEL_WARNINGS
 
 /**
- * @brief Initializes each slot in the bucket storage to contain `value`.
+ * @brief Initializes each slot in the flat storage to contain `value`.
  *
- * @tparam BucketT Bucket type
+ * @tparam T Value type
  *
- * @param buckets Pointer to flat storage for buckets
- * @param n Number of input buckets
+ * @param slots Pointer to flat storage for slots
+ * @param n Number of input slots
  * @param value Value to which all values in `slots` are initialized
  */
-template <typename BucketT>
-CUCO_KERNEL void initialize(BucketT* buckets,
-                            cuco::detail::index_type n,
-                            typename BucketT::value_type value)
+template <typename T>
+CUCO_KERNEL void initialize(T* slots, cuco::detail::index_type n, T value)
```

Comment:
We could replace this now with a `thrust::fill`

Reply:
Great point. I replaced it with `cub::DeviceFor::Bulk` since thrust algorithms may internally perform raw `cudaMalloc` and `cudaFree` operations for temporary storage.